Src/EB/AMReX_eb_to_pvd.F90

```diff
@@ -627,21 +626,25 @@ subroutine amrex_eb_grid_coverage (myID, dx, lo, hi, flag, fglo, fghi)&
 
 contains
 
-  subroutine data_array(lo, lnodes, ldx)
+  subroutine data_array(problo, lo, lnodes, ldx)
 
     implicit none
 
     integer,          intent(in) :: lo, lnodes
+    real(amrex_real), intent(in) :: problo
     real(amrex_real), intent(in) :: ldx
 
     integer :: llc
     real(amrex_real), allocatable :: lines(:)
+    real(amrex_real)              :: grid_start 
 
     if(allocated(lines)) deallocate(lines)
     allocate(lines(0:lnodes))
 
-    do llc=0, lnodes
-       lines(llc) = real(lo + llc)*ldx
+    grid_start = problo + real(lo)*ldx
```

Comment:
I think `real` here (and three lines below probably) should be `dble(lo)` or `real(lo,amrex_real)`.